transport/server.cc

```diff
@@ -188,9 +188,13 @@ event::event_type parse_event_type(const sstring& value)
     }
 }
 
-cql_sg_stats::cql_sg_stats()
+cql_sg_stats::cql_sg_stats(db::maintenance_socket_enabled used_by_maintenance_socket)
     : _cql_requests_stats(static_cast<uint8_t>(cql_binary_opcode::OPCODES_COUNT))
 {
+    if (used_by_maintenance_socket) {
```

Comment:
@avikivity looks like metrics are disabled on the maintenance socket. I think it makes sense, having combined metrics would most likely require a larger refactor, but I don't think there is a point in having metrics on maintenance socket -- one shouldn't run any serious workloads on it.